.githooks/pre-commit

```diff
@@ -6,7 +6,7 @@ exec 1>&2
 find src unittests -type f \( -name '*.cpp' -o -name '*.h' -o -name '*.ipp' \) -print0 | xargs -0 clang-format -i
 
 # check how many lines differ
-lines=$(git diff | wc -l)
+lines=$(git diff src unittests | wc -l)
```

Comment:
This change was not planned but i realised my mistake from yesterday and decided to fix it here and now